### ISSUE TITLE
Installing the game empties the selectd folder first

### ISSUE THREAD

**Issue by madedhun** [remyroy/CDDA-Game-Launcher#194]
I just downloaded the launcher and install the game to my "games" directory.
It just deleted every other game I had installed (15+) along with their saves.... so yeah...

Please review your installation mechanics to eihter require an empty directory or 'ASK' before deleting anything.

**Comment by remyroy** [comment]
You should not use the launcher in a directory that is contains other stuff. This might need to be explained in greater details.

**Comment by DAOWAce** [comment]
Gotta agree with OP.

Very bad design to forcibly delete everything in a folder without warning.

There's no real instructions with the launcher either; just download the exe and run it; it automatically uses its running location as the main game directory.

Shouldn't be too hard to prompt about selecting an empty folder, right?  Or at the very least force it into its own subdirectory by default.

This also extends to the update mechanism too; it wipes the previous_version folder.  I lost my older 0.C game from a few years ago.

An update can also error, be it when restoring previous version or actually updating.  It will leave the game in a broken state (mine emptied the config folder).  The program sometimes doesn't move some folders to the previous_version one (notably the save/backups).  Attempting to force another update makes it wipe the folder.

Basically, the tool isn't **safe**.  It needs to be made safe.

**Comment by remyroy** [comment]
Files are moved in a sub directory called previous_version so nothing is deleted during the installation or the update.

When I first publicized the launcher, I included [many instructions](https://www.reddit.com/r/cataclysmdda/comments/3yv7d7/cdda_game_launcher_automatic_updates_and_more/). If you came here from a different source, they did a bad job at instructing you.

Some safeguards should be added. I'll see what I can do to make this better.

**Comment by DAOWAce** [comment]
Might be best to copy your original post into the Github readme, as that's where I found my link to the software.  It's missing the FAQ here.   Though some people will still ignore it regardless..
